atd-vze/src/views/Crashes/crashGridTableParameters.js

```diff
@@ -57,6 +63,7 @@ export const crashGridTableColumns = {
     sortable: false,
     label_table: "Unit Description",
     type: "String",
+    hidden: true,
```

Comment:
👍 it makes sense, that way we can have the unit description in the data, but not show it in the UI